libfabric.spec

```diff
@@ -51,12 +48,12 @@ BuildRequires: autoconf, automake, libtool
 
 %ifarch x86_64
 %if 0%{?suse_version} >= 01315 || 0%{?rhel} >= 7
-%global configopts --enable-sockets --enable-verbs --enable-usnic --disable-static --enable-psm2 --disable-efa --without-gdrcopy
+%global configopts --disable-sockets --enable-verbs --enable-usnic --disable-static --enable-psm2 --disable-efa --without-gdrcopy --disable-opx --enable-tcp 
```

Comment:
why are we disabling sockets? we should keep sockets available for a backup. if anytihng we should do --diable-psm2 and --disable-psm3 as we do not support those at all.

Reply:
I just update to enable both sockets and tcp. "The sockets provider has been deprecated in favor of the tcp, udp, and utility providers" from libfabric. I thought it would be good to force tcp is used when verbs and psm are not available.

Reply:
tcp;ofi_rxm support is still rather new, compared to verbs and sockets. as such, at least for now, i would prefer to have this last backup option to use sockets if everything else fails. thanks:)

Reply:
Understood. Thank you!